### DISCUSSION TITLE
App for android tv

### DISCUSSION THREAD

**Discussion by NimuthuGanegoda** [kodjodevf/mangayomi#92] [Ideas]
Would you be able to issue a .apk for android tv ?

**Comment by kodjodevf** [comment]
I don't think so at the moment because the package that the App uses to play videos is not compatible with Android TV, although I will try to do it to see the result as soon as I have time I will share .apk .

**Reply by Magededward** [reply]
hello dear!

I wanted to check if you do have any update on the Android TV app? or have any testing as it would be great!

**Reply by yxxyun** [reply]
https://github.com/media-kit/media-kit/issues/614#issuecomment-2380777827
`const configuration = VideoControllerConfiguration(vo: 'mediacodec_embed', hwdec: 'auto');

videoController = VideoController(player, configuration: configuration);`
